File: kombarika/com/district/test/controller/ActiviteController.java
package com.district.test.controller;


import com.district.test.repository.ActiviteRepository;
import com.district.test.entity.Activite;
import org.springframework.http.*;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.*;


@RestController
@CrossOrigin("*")
@RequestMapping(path = "activite")
public class ActiviteController
 {

	@Autowired
	private ActiviteRepository repository;


	@PostMapping()
	public ResponseEntity<Activite> save(@RequestBody Activite activite){
	 	return ResponseEntity.ok(repository.save(activite));
	}
	@PutMapping()
	public ResponseEntity<Activite> update(@RequestBody Activite activite){
	 	return ResponseEntity.ok(repository.save(activite));
	}
	@DeleteMapping()
	public void delete(@RequestBody Activite activite){
	 	return ResponseEntity.ok(repository.delete(activite));
	}
	@GetMapping()
	public ResponseEntity<Iterable<Activite>> findAll(){
	 	return ResponseEntity.ok(repository.findAll());
	}





}

File: kombarika/com/district/test/controller/ActiviteController.cs
namespace com.district.test.controller;


using System.Diagnostics;
using Microsoft.AspNetCore.Mvc;
using com.district.test.repository.ActiviteContext;
using com.district.test.entity.Activite;


[ApiController]
[Route('[activite]')]
public class ActiviteController
 {

	private readonly Ilogger<ActiviteController> _logger;


	[HttpPost]
	public ActionResult<Activite> save([FromBody] Activite activite){
	 	_context.MyItems.Add(value);
		_context.SaveChanges();
		return Ok();
	}
	[HttpPut]
	public ActionResult<Activite> update([FromBody] Activite activite){
	 	var temp = activite;
		_context.SaveChanges();
		return Ok();
	}
	[HttpDelete]
	public void delete([FromBody] Activite activite){
	 	_context.MyItems.Remove(activite);
		_context.SaveChanges();
		return Ok();
	}
	[HttpGet]
	public ActionResult<IEnumerable<Activite>> findAll(){
	 	return Ok(_context.activite.ToList());
	}

	public ActiviteController(ActiviteDbContext context) { _context = context; }



}
